.gitignore

```diff
@@ -29,3 +29,4 @@ config/application.yml
 
 # Ignore mock mvi responses
 /config/mvi_schema/mock_mvi_responses.yml
+.idea/
```

Comment:
I can't decide how I feel about editor specific ignore files. On the one hand, you can just put them in your personal root gitignore so as not the clutter up the file. On the other hand, why risk that someone else on the team will forget to ignore their editor specific files? In this case, we don't have DS_Store in the gitignore, but probably the majority of the team is on a Mac. It's hard to draw bright line. 😐